client/state/account-recovery/settings/actions.js

```diff
@@ -229,7 +233,7 @@ export const validateAccountRecoveryPhone = code => dispatch => {
 	return wpcom
 		.undocumented()
 		.me()
-		.validateAccountRecoveryPhone( code )
+		.validateAccountRecoveryPhone( replace( code, ' ', '' ) )
```

Comment:
This replaces only first occurrence
Not sure why should we use that instead of `String.prototype.replace` ?

![screen shot 2018-03-07 at 16 02 03](https://user-images.githubusercontent.com/326402/37096342-03a53432-2221-11e8-8bee-2aefb3a1c6e8.png)

Reply:
> This replaces only first occurrence

Good catch, I'll fix that.

> Not sure why should we use that instead of String.prototype.replace ?

I've opted to use Lodash's `replace()` since it handles `undefined` and `null` values for `code` gracefully.